BuildAndTest.proj

```diff
@@ -75,13 +76,17 @@
 
       <!-- MakeConst unit tests tracked by https://github.com/dotnet/roslyn/issues/5918 -->
       <TestAssemblies Condition="'$(Test64)' != 'true'" 
-                      Include="$(OutputDirectory)\**\$(IncludePattern)" />
+                      Include="$(OutputDirectory)\$(IncludePattern)" />
```

Comment:
Given that you now don't run tests outside of the root or in the core clr folder, can we break the build if there are tests found outside of it?